artiq/coredevice/ad9910.py

```diff
@@ -242,6 +242,10 @@ def read32(self, addr):
 
         :param addr: Register address
         """
+        return self.read32_impl(addr)
+
+    @kernel
+    def read32_impl(self, addr):
```

Comment:
This should probably in the (to come) suservo commit, with an explanatory comment.